model/src/test/kotlin/MergedAnalyzerResultTest.kt

```diff
@@ -26,31 +26,35 @@ import java.io.File
 
```

Comment:
Commit message: "This fixes some compiler warnings." Actually, these should have been just hints (from the linter), as we fail on compiler warnings.

Reply:
Done.